matter/si91x/siwx917/BRD4325x/autogen/sl_component_catalog.h

```diff
@@ -2,10 +2,15 @@
 #define SL_COMPONENT_CATALOG_H
 
 // APIs present in project
+#define SL_CATALOG_BTN0_PRESENT
+#define SL_CATALOG_BUTTON_PRESENT
+#define SL_CATALOG_BUTTON_BTN0_PRESENT
+#define SL_CATALOG_BTN1_PRESENT
+#define SL_CATALOG_BUTTON_BTN1_PRESENT
 #define SL_CATALOG_CMSIS_OS_COMMON_PRESENT
+#define SL_CATALOG_DEVICE_INIT_NVIC_PRESENT
 #define SL_CATALOG_EMLIB_CORE_PRESENT
 #define SL_CATALOG_EMLIB_CORE_DEBUG_CONFIG_PRESENT
 #define SL_CATALOG_FREERTOS_KERNEL_PRESENT
-#define SL_CATALOG_KERNEL_PRESENT
```

Comment:
Is there a reason to remove this define? I don't see a direct impact in the Matter apps, but it will affect the behavior of the new Generator Firmware used by CPMS provisioning

Reply:
Ohh thought since this was not affect, I can remove this. Will readd this define

Reply:
Readded the define on the latest commit